NewForce-Cohort-9/tabloid-full-stack-full-stack-force
# Fw ticket19 -View Comments

Description
Added a feature to view comments for a specific post. [Trello Ticket #19](https://trello.com/c/V8SrJKyg/19-19-view-a-posts-comments)


Type of Change
-Added new, non-breaking feature.


Testing Instructions
1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout fw-ticket19
5. If you don't see the most recent version of my files, git pull origin fw-ticket19
6. If you haven't created the Tabloid database, please do so.
7. Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-full-stack-force/blob/main/SQL/01_Tabloid_Create_DB.sql)
8. Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-full-stack-force/blob/main/SQL/02_Tabloid_Seed_Data.sql)
9. Inside of Visual Studio (purple), click the execute with debugger solid green button to launch the app
10. In your terminal, navigate to where your package.json file is in you client folder. Run "npm start" in your terminal.
11. If not logged in, you may login with any user in the database. I have been using foo@bar.com for the email parameter. 
12. Leave the password text box blank. Click "Login".
13. Click on “Posts” in the nav bar, this will redirect you to the /posts page.
14. You should be redirected to a page that lists all posts.
15. Click on a post title, it should direct you to the post's details.
16. You will see details of the posts and a "View Comments" button.
17. Click on the "View Comments" button.
18. You should be redirected to a page that lists all comments for that post sorted by most recent first.
19. You will see a link back to post and the following fields for each comment:
-Subject
-Content
-Author
-Creation Date


Checklist:
[x] My code follows the style guidelines of this project
[x] I have performed a self-review of my code
[x] I have commented my code, particularly in hard-to-understand areas
[x] My changes generate no new warnings or errors
[x] I have added test instructions that prove my fix is effective or that my feature works
